packages/app/src/app/components/Preview/DevTools/Console/index.tsx

```diff
@@ -104,7 +109,7 @@ class ConsoleComponent extends React.Component<StyledProps> {
           if (aggregatedResults) {
             const { summaryMessage, failedMessages } = aggregatedResults;
             this.addMessage('log', [summaryMessage]);
-            failedMessages.forEach(t => {
+            failedMessages.forEach((t) => {
```

Comment:
eyh... what is `t`? ;-) a message?

Reply:
Ahh, this turned up in the diff because of prettier 😂 . I can find a better name.